content/ita/careers.md

```diff
@@ -12,17 +12,22 @@ twitterlargeimage: "/images/careers/banner.png"
 
 ![Facile.it - lavora con noi!](/images/careers/banner.png)
 
-**Facile.it** è in costante crescita e siamo sempre alla ricerca di programmatori da aggiungere al nostro team. Questa proposta è rivolta a sviluppatori di grande esperienza, che credono nell’innovazione e sono sempre attenti alle nuove tecnologie e non vedono l’ora di metterle in pratica. Siamo un gruppo di lavoro dinamico, giovane e divertente. Siamo in totale quasi 80 programmatori e gestiamo numerosi progetti con differenti stack tecnologici.
+**Facile.it** is constantly growing and we are always looking for programmers to join our team.
+This proposal is aimed at highly experienced developers who believe in innovation and are always attentive to new technologies and can't wait to put them into practice. We are a dynamic, young and fun team.
+We are more than 100 programmers and manage numerous projects with different technology stacks.
 
-# Cosa offriamo in Facile.it:
+# What we offer at Facile.it:
 
- * Assunzione a tempo indeterminato con RAL in base alla tua esperienza + fondo salute integrativo
- * Possibilità di crescita economica in base alle nuove competenze acquisite
- * Un venerdì al mese dedicato alla formazione e al team building
- * Biblioteca di libri di tecnologia con più di 100 titoli
- * Partecipazione ad eventi di tecnologia a nostro carico (SymfonyCon, phpDay, Codemotion, etc.)
- * Ufficio in centro a Milano (MM3 Missori) con sala relax, sofà, ping-pong e calcetto
- * Convenzioni (bar, ristoranti, palestre e molto altro), Ticket Restaurant, caffè, acqua e frutta fresca ogni giorno
- * Fare il lavoro che ami in una realtà italiana di successo con un top team al tuo fianco
+* Permanent employment with remuneration commensurate to your experience level, and supplementary health fund
+* A young and dynamic IT team with an open mind to new technologies and processes
+* Personalized introduction and growth plans
+* 1 Friday per month dedicated to learning, sharing and studying with the whole IT department
+* Personal study budget
+* Smart Working 4 days a week or the possibility of working full remote
```

Comment:
"Smart Working" meaning "work from home" is an incorrect use of the term in Italian.
I would write as follows:
Remote work 4 days per week or 100% of the time (based on agreement).